### PR TITLE
:seedling: TK-1986 : Updated Log Collector README

### PR DESCRIPTION
**Pull request by pallav-trilio**

### CODE REVIEW

File: docs/log-collector/README.md [hunk 1]
```diff
@@ -1,26 +1,57 @@
 # k8s-triliovault Log Collector
-This is a python **pip** module that collects the information mainly yaml configuration and logs from k8s cluster for debugging k8s-triliovault application.
 
-## Requirements
-1. python >= 3.6
+Log collector let you define what you need to log and how to log it by collecting the the logs and events of Pod. Pod Logs can help you understand what is happening inside your application. The logs are particularly useful for debugging problems and monitoring cluster activity, alongside the metadata of all resources related to TrilioVault as either namespaced by providing namespaces name separated by comma or clustered from k8s cluster for debugging k8s-triliovault application. It also collects the CRDs yaml related to TVK and zip them.
```
Comment (msurudwad) [inline docs/log-collector/README.md:3]: `Log collector let you define what you need to log and how to log it` how? It seems wrong to me. Log collector just collects logs and events of resources.

File: docs/log-collector/README.md [hunk 2]
```diff
@@ -1,26 +1,57 @@
...
+- Installation:
+
+
+    kubectl krew install tvk-plugins/tvk-log-collector
+
+
+- Upgrade:
+
+
+    kubectl krew upgrade
```
Comment (msurudwad) [inline docs/log-collector/README.md:29]: update it to >>   kubectl krew upgrade tvk-log-collector

File: docs/preflight/README.md [hunk 3]
```diff
@@ -39,16 +39,48 @@ The following checks included in preflight:
...
+- Installation:
+
+
+    kubectl krew install tvk-plugins/tvk-preflight
+
+
+- Upgrade:
+
+
+    kubectl krew upgrade
```
Comment (msurudwad) [inline docs/preflight/README.md:66]: update it to >> `kubectl krew upgrade tvk-preflight `

**Review by msurudwad — CHANGES_REQUESTED** [review]
1. Change description of preflight plugin in `docs/preflight/README.md`
2. Add section of `supported OS and Architecture` for both plugins
3. Improve the doc formatting like command highlighting, headline formats.

File: docs/log-collector/README.md (hunk 1 above)
Comment (pallav-trilio) [reply docs/log-collector/README.md:3]: How to log i mean the level of log such as info error warn

File: docs/preflight/README.md (hunk 3 above)
Comment (pallav-trilio) [reply docs/preflight/README.md:66]: resolved

File: docs/log-collector/README.md (hunk 2 above)
Comment (pallav-trilio) [reply docs/log-collector/README.md:29]: resolved

File: docs/log-collector/README.md (hunk 1 above)
Comment (pallav-trilio) [reply docs/log-collector/README.md:3]: resolved

**Pull request merged**